app/controllers/application_controller.rb

```diff
@@ -5,4 +5,9 @@ class ApplicationController < ActionController::Base
   def current_user
     @current_user ||= User.find_by(id: session[:user_id])
   end
+
+  # Returns whether hcaptcha is enabled by checking if ENV variables are set
+  def hcaptcha_enabled?
+    (ENV['RECAPTCHA_SITE_KEY'].present? && ENV['RECAPTCHA_SECRET_KEY'].present?)
+  end
```

Comment:
This is fine for now, but I'm thinking we move this to a concern called `EnvSettings` or something later on